documentation/run-strategies-template.ejs

```diff
@@ -25,8 +25,10 @@ You can also [create your own](#create-your-own).
 <? }); ?>
 
 <a name="create-your-own"></a>
-#### Create your own
+#### Create Your Own
 
-You can create your own strategy by passing in a function in the `strategy` parameter to the `F.manager.RunManager()` instantiation call. 
+You can create your own strategy by:
+
+* Passing in a function in the `strategy` parameter to the `F.manager.RunManager()` instantiation call. Strategy functions must return a boolean value for whether or not to create a new run. 
+* Optionally, passing in a `strategyOptions` object with any configuration options that your strategy function may use. 
```

Comment:
"pass in", not "passing in" right? I'd reword as:

Some strategies have options you can specify through `strategyOptions` in the RunManager. If you create your own strategy, this will be passed in to your strategy function constructor.